### ISSUE TITLE
(Probable bug) Incidence rate has 'restrict to the same visit occurrence'

### ISSUE THREAD

**Issue by gowthamrao** [OHDSI/Atlas#650]
### Expected behavior
I dont think it is possible to restrict to the same visit_occurrence_id as the condition_occurrence_id under incidence rate strata specification

### Actual behavior
It is possible to select restrict to same visit_occurrence_id in strata specification of incidence rate
![image](https://user-images.githubusercontent.com/13936600/38886013-a8ea9086-4242-11e8-9c89-b9cbdff24fcf.png)

### Steps to reproduce behavior
Probably won't run or is a bug

**Comment by chrisknoll** [comment]
This will run _if_ the cohort episodes query returns a visit_concept_id of _null_ before feeding it over to stratification.  Restricting to same visit occurrence at this level does not make sense (cohort episodes don't have visit associations) however if you were to create a nested criteria, you would want to show the 'restrict visit' option.  

I forget how we control the visibility of the checkbox...I think we check the 'wrapped' criteria type (in the above and if the wrapped criteria type has a visit (ie condition does, condition era does nto) then we show the checkbox.  But what is actually needed is a check for both sides: the wrapp**ed** criteria supports a visit, and the wrapp**ing** context also has to have a context of a visit.

I'll check to see if the IR code emits a 'visit_concept_id' in the query (as null), and if so, then the only thing that will ahpepn is that if you try to restrict to the same visit at the top level of a strata criteria to the event (cohort episode start) nothing will match that and you'll have no records in the strata.

**Comment by chrisknoll** [comment]
Nope! it's a bug, the cohort events are not setting NULL to the visit_id so it will break. Thank you for the report!
